library/gcse/src/gcse_page_map.e

```diff
@@ -0,0 +1,38 @@
+note
+	description: "Represent a google page map"
+	date: "$Date: 2015-10-09 08:11:07 -0300 (vi., 09 oct. 2015) $"
+	revision: "$Revision: 97973 $"
+	EIS: "name=PageMaps", "src=https://developers.google.com/custom-search/docs/structured_data#pagemaps", "protocol=url"
+
+class
+	GCSE_PAGE_MAP
```

Comment:
I guess this is not yet supported for now, right?

maybe the commented lines could be in a top indexing note.